radlab-ui/webapp/src/routes/Deployments.tsx

```diff
@@ -36,6 +40,39 @@ const Deployments: React.FC<DeploymentsProps> = () => {
   const [deploymentTab, setDeploymentTab] = useState(
     isAdmin ? DEPLOYMENT_TAB.ALL : DEPLOYMENT_TAB.MINE,
   )
+  const filteredDeployments = deploymentStore(
+    (state) => state.filteredDeployments,
+  )
+  const setDeployments = deploymentStore((state) => state.setDeployments)
+  const [modules, setModules] = useState<IModule[] | null>(null)
+
+  const deletedAt = "DELETED"
+
+  const activeStatuses = [
+    DEPLOYMENT_STATUS.SUCCESS,
+    DEPLOYMENT_STATUS.PENDING,
+    DEPLOYMENT_STATUS.QUEUED,
+    DEPLOYMENT_STATUS.FAILURE,
+    DEPLOYMENT_STATUS.EXPIRE,
+    DEPLOYMENT_STATUS.CANCELLED,
+    DEPLOYMENT_STATUS.INTERNAL_ERROR,
+    DEPLOYMENT_STATUS.STATUS_UNKNOWN,
+    DEPLOYMENT_STATUS.TIMEOUT,
+    DEPLOYMENT_STATUS.WORKING,
+    deletedAt,
+  ]
+
+  const fetchModules = async () => {
+    await axios
+      .get(`/api/github/modules`)
+      .then((res) => {
+        setModules(res.data)
+      })
+      .catch((error) => console.error(error))
```

Comment:
We need to handle this error more than just log it

Reply:
By setting alert? Like `setAlert()=>{}`